### ISSUE TITLE
When navigating mention suggestions with arrow keys keyDown events should not propogate

### ISSUE THREAD

**Issue by tommoor** [draft-js-plugins/draft-js-plugins#576]
First of all - thanks for the great, high quality, customizable plugins here - nice work team 👍 

When the up / down arrow keys are used to navigate the list of mention suggestions the plugin should use `preventDefault` / `stopPropagation` on the event to prevent it from bubbling up to the containing application.

For example, in our app the keys are bound to navigating another list and when the mentions popover is open then both lists move…

**Comment by nikgraf** [comment]
good point @tommoor - wanna add a PR? The earliest I would get to it is post 2.0 launch in 1-2 weeks …

**Comment by githubflub** [comment]
Is this still an issue? 

I wrote a mention-like plugin, and if it's active on the line above or below an image (draft-js-image-plugin), the draft-js-focus-plugin will move the cursor around and select the image. Ideally, it should not.

**Comment by artemplv** [comment]
> Is this still an issue?
> 
> I wrote a mention-like plugin, and if it's active on the line above or below an image (draft-js-image-plugin), the draft-js-focus-plugin will move the cursor around and select the image. Ideally, it should not.
@githubflub hi! Were you be able to solve this kind of behaviour back then?

**Comment by githubflub** [comment]
@artemplv This was such a long time ago, but I believed that I used a global variable to handle this. The plugins would always run in the same order, so by setting some variable in one plugin like 

`window.dont_propagate_draftjs_plugin = true`

I was able to stop any subsequent plugin from running. Also, I think in order to do this, I had to copy each plugin's source code, meaning not use the packaged code.
